packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m

```diff
@@ -10,7 +10,7 @@
 static FlutterError *getFlutterError(NSError *error) {
   if (error == nil) return nil;
 
-  return [FlutterError errorWithCode:[NSString stringWithFormat:@"Error %ld", error.code]
+  return [FlutterError errorWithCode:[NSString stringWithFormat:@"Error %ld", (long)error.code]
```

Comment:
LGTM!
Another way instead of force casting would be always use NSObjects
such as 
```suggestion
  return [FlutterError errorWithCode:[NSString stringWithFormat:@"Error %@", @(error.code)]
```

But I'm ok with whichever you prefer.

Reply:
Thanks! Good to know!